manifests/agent.pp

```diff
@@ -228,6 +237,8 @@
   $listenip                               = $zabbix::params::agent_listenip,
   $startagents                            = $zabbix::params::agent_startagents,
   $serveractive                           = $zabbix::params::agent_serveractive,
+  Enum['running','stopped'] $service_state  = $zabbix::params::agent_service_state,
```

Comment:
Can you instead use the stdlib datatype for this:
https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/ensure/service.pp#L1